spinnaker/resource_project.go

```diff
@@ -0,0 +1,99 @@
+package spinnaker
+
+import (
+	"strings"
+
+	"github.com/armory-io/terraform-provider-spinnaker/spinnaker/api"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceProject() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
```

Comment:
do you plan on adding types for `applications`, `pipelines` and `clusters` as well?

Reply:
Yeah, that's what I had in mind since that'd enable a full declaration of a project spanning the applications/pipelines/clusters.